LICENSE

```diff
@@ -0,0 +1,28 @@
+MIT License
+
+Copyright (c) 2020 Dhairya Gandhi
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+The code in this repository - the C wrapper and the parts in Julia - are
+licensed under the MIT license. The PyTorch library installed as a consequence
+of installing this library, via the Torch_jll artifact is separately licensed
+by PyTorch. The source code can be found at
+https://github.com/pytorch/pytorch/blob/master/LICENSE and its license can be
+found at https://github.com/pytorch/pytorch/blob/master/LICENSE
```

Comment:
Better to copy the contents of the LICENSE here.

Reply:
Added the contents here